### DISCUSSION TITLE
LB4 update has broke AJV Validation

### DISCUSSION THREAD

**Discussion by Habbiton** [loopbackio/loopback-next#8346] [Q&A]
Hi there!

I've upgrade LB4 and now, login has broken with this error message:
Request POST /login failed with status code 500. Error: strict mode: unknown keyword: "email"

I thinks it's related to [THIS](https://ajv.js.org/v6-to-v8-migration.html) but i'm not sure where to start with.

PD: Updated dependencies:

    "@loopback/authentication": "^7.3.1 -> ^8.0.0",
    "@loopback/authentication-jwt": "^0.9.1 -> ^0.10.0",
    "@loopback/authorization": "^0.9.1 -> ^0.10.0",
    "@loopback/boot": "^3.4.1 -> ^4.0.0",
    "@loopback/context": "^3.17.0 -> ^4.0.0",
    "@loopback/core": "^2.16.1 -> ^3.0.0",
    "@loopback/openapi-v3": "^5.3.1 -> ^7.0.0",
    "@loopback/repository": "^3.7.0 -> ^4.0.0",
    "@loopback/rest": "^9.3.1 -> ^11.0.0",
    "@loopback/rest-explorer": "^3.3.1 -> ^4.0.0",
    "@loopback/service-proxy": "^3.2.1 -> ^4.0.0",
    "@loopback/build": "^6.4.1 -> ^8.0.0",
    "@loopback/eslint-config": "^10.2.1 -> ^12.0.0",
    "@loopback/testlab": "^3.4.1 -> ^4.0.0",
    "@typescript-eslint/eslint-plugin": "^4.26.1 -> ^4.33.0",
    "@typescript-eslint/parser": "^4.26.1 -> ^4.33.0",
    "eslint": "^7.28.0 -> ^7.32.0",
    "prettier": "2.3.2 -> ^2.4.1",
    "source-map-support": "^0.5.19 -> ^0.5.20",
    "typescript": "~4.3.2 -> ~4.5.2"

Thank you

**Comment by mrmodise** [comment]
Hi,

What is your npm and node version?
